### ISSUE TITLE
Illumination Slider Step Size

### ISSUE THREAD

**Issue by kmcleaster** [EtalumaSupport/LumaViewPro#180]
For Brightfield and the two Phase Contrast modes, the current stepsize of 5mA is too large. It is also difficult to advance the slider by only 5mA. I also discovered that the minimum BF current setting is 4mA. It may be necessary to have individual characteristics for each channel's illumination controls. I also found that the entry windows seem to be particularly laggy and difficult to set on this Feb 7, 2023 codebase.

**Comment by EtalumaSupport** [comment]
Currently you must type "2" into both illumination and exposure to get a greyscale BF image, this is very uncomfortable.  1 is too dark and 3 is too bright

**Comment by tnerdi** [comment]
@EtalumaSupport Is the issue that you have to enter the same value many times over? Would you like "2" to be the default value for both illumination and exposure?

**Comment by EtalumaSupport** [comment]
Yes, the first three channels shine light directly into the objective while the second three channels project light at the sample and collect the glow or reflection, which is much dimmer.   So the lower illumination settings (say 1-10) should be spread over the whole range of the slider.  Right now the slider jumps by 5s, too coarse too.
